src/FRP/Event.purs

```diff
@@ -107,3 +108,11 @@ foreign import subscribe
    . Event a
   -> (a -> Eff (frp :: FRP | eff) r)
   -> Eff (frp :: FRP | eff) (Eff (frp :: FRP | eff) Unit)
+
+-- | Create an event and a function which supplies a value to that event.
+foreign import create
```

Comment:
I would prefer to put this in an `Internal` module. We could always put `fix` in that module as well and then reexport it here.

Reply:
But I think we would have to define `Event` there, to avoid circular imports ... and instances, too. Unless we want some `unsafeCoerce` from the `EffFn` representation.

Reply:
Oh, I see. Yes, we would need to add a `Types` module or something. Ok, I'm fine with this then, thanks.